modules/core/include/opencv2/core/persistence.hpp

```diff
@@ -315,6 +315,7 @@ class CV_EXPORTS_W FileStorage
         APPEND      = 2, //!< value, open the file for appending
         MEMORY      = 4, //!< flag, read data from source or write data to the internal buffer (which is
                          //!< returned by FileStorage::release)
+                         //!<
```

Comment:
I believe it makes sense merge these lines into one instead.

3.4 version is [not broken](https://docs.opencv.org/3.4/da/d56/classcv_1_1FileStorage.html#a973e41cb75ef6230412a567723b7482d).
4.x version is [broken](https://docs.opencv.org/4.x/da/d56/classcv_1_1FileStorage.html#a973e41cb75ef6230412a567723b7482d).

Patch "live" changes: [enum cv::FileStorage::Mode](http://pullrequest.opencv.org/buildbot/export/pr/21582/docs/da/d56/classcv_1_1FileStorage.html#a973e41cb75ef6230412a567723b7482d)

Reply:
I'm happy to make it single-line, however this creates a 139 character long line that appears to go against the [coding style guide](https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure). Please advise if this is an allowed exception.

No issue changing the target branch if necessary. That should be a clean patch.

Reply:
It is OK to exceed lines length with comments.

Reply:
Okay, I've made this correction. Marking comment as resolved.

Reply:
Could you please revert target branch change and target to 3.4?

There is similar problem on 3.4 branch. But due to some reason it doesn't trigger buggy doxygen behavior.

Reply:
BTW, It makes sense to put multi-line doc string as multi-line C++ comment:

```
        MEMORY      = 4, /**< flag, read data from source or write data to the internal buffer
                              (which is returned by FileStorage::release) */

```

Reply:
Switch back to 3.4. Made this multiline C++ comment, retaining the original line break location.